build/docker/zap_common.py

```diff
@@ -254,6 +254,18 @@ def zap_ajax_spider(zap, target, max_time):
     logging.debug('Ajax Spider complete')
 
 
+def zap_active_scan(zap, target, policy):
+    logging.debug('Active Scan ' + target + ' with policy ' + policy)
+    ascan_scan_id = zap.ascan.scan(target, recurse=True, scanpolicyname=policy)
+    time.sleep(5)
+
+    while(int(zap.ascan.status(ascan_scan_id)) < 100):
+        logging.debug('Active Scan progress %: ' + zap.ascan.status(ascan_scan_id))
+        time.sleep(5)
+    logging.debug('Active Scan complete')
+    logging.debug(zap.ascan.scan_progress(ascan_scan_id))
```

Comment:
@kingthorin both lines (263 and 264) are indented the same amount and so are both in the while loop. So I _think_ its as its supposed to be...